src/docs/multiomics_submission.md

```diff
@@ -0,0 +1,45 @@
+
+When multiple samples belong to the same project, the BioSamples documentation recommends adding the name of the project to the `Project` field of the `Attribute` table to make the samples collectively searchable.
+
+A useful feature of the BioSamples repository is that samples generated from the same specimen can be linked by declaring relationships between BioSamples IDs. This way, the user can link the specimen metadata with the sample(s) derived from it. A simplified example is depicted in [Figure 1](@multiomics_schema). In reality, the relationships across samples are highly dependent on the experimental design of each particular project. It is therefore advisable to build the planning of your sample relationships into the process of designing your project and experiments. Other relationships between samples in [BioSamples are supported](https://www.ebi.ac.uk/biosamples/docs/guides/relationships), such as “same as” or “has member” relationships. You should use the available relationships to best represent your dataset, while trying to keep the overall sample organisation as simple as possible.
+
+[Figure 1. A simplified example of sample relationships established on the BioSamples repository for a multi-omics project. Bold font indicates unique BioSamples ID](images/multiomics/schema.png){#multiomics_schema}
+
+## A case use example: HoloFood project
+
+The HoloFood project adopts a 'hologenomic' approach to enhance the efficiency of food production systems. This approach focuses on comprehending the biomolecular and physiological processes influenced by integrating feed additives and innovative sustainable feeds in farm animals. Holofood leverages various cutting-edge technologies to explore the intricate interactions between animals and their respective gut microbiomes on multiple molecular levels. Samples and data generated by the HoloFood project can be accessed through the [Holofood Data Portal](https://www.holofooddata.org/).
+
+Multi-omics data generated from both animal host and microbiome, along with physiochemical and physiological data, have been deposited into the BioSamples database and other relevant repositories. To avoid multiple levels of nested relationships, the authors opt for assigning a BioSamples ID to each animal and linking it to the corresponding biological samples through ‘DERIVED FROM’ relationships. For instance, animal [`SAMEA112905287`](https://www.ebi.ac.uk/biosamples/samples/SAMEA112905287) has 8 derived samples with external links to other repositories when relevant (see [Figure 2](@SAMEA112905287_example)). Examples of samples with “derived from” relationships to animal `SAMEA112905287` are host genomics (linked to the ENA), metagenomic assembly (linked to the ENA), and metabolomic (linked to MetaboLights). Other animal samples can also have associated transcriptomics (ENA), metatranscriptomics (ENA), inflammatory markers (stored in BioSamples), or histological data (stored in BioSamples).
```

Comment:
```suggestion
Multi-omics data generated from both animal host and microbiome, along with physiochemical and physiological data, have been deposited into the BioSamples database and other relevant repositories. To avoid multiple levels of nested relationships, the authors opt for assigning a BioSamples ID to each animal and linking it to the corresponding biological samples through ‘DERIVED FROM’ relationships. For instance, animal [`SAMEA112905287`](https://www.ebi.ac.uk/biosamples/samples/SAMEA112905287) has 8 derived samples with external links to other repositories when relevant (see (@SAMEA112905287_example)). Examples of samples with “derived from” relationships to animal `SAMEA112905287` are host genomics (linked to the ENA), metagenomic assembly (linked to the ENA), and metabolomic (linked to MetaboLights). Other animal samples can also have associated transcriptomics (ENA), metatranscriptomics (ENA), inflammatory markers (stored in BioSamples), or histological data (stored in BioSamples).
```